test/uk/gov/hmrc/thirdpartyapplication/connector/EmailConnectorSpec.scala

```diff
@@ -308,5 +308,25 @@ class EmailConnectorSpec extends ConnectorSpec {
 
       await(connector.sendChangeOfApplicationName(requesterName, previousAppName, newAppName, recipients))
     }
+
+    "send change of application details" in new Setup {
+      val requesterName                           = "bob@example.com"
+      val applicationName                         = "App name"
+      val fieldName                               = "privacy policy URL"
+      val previousValue                           = "https://example.com/previous-privacy-policy"
+      val newValue                                = "https://example.com/new-privacy-policy"
+      val expectedParameters: Map[String, String] = Map(
+        "requesterName"   -> requesterName,
+        "applicationName" -> applicationName,
+        "fieldName"       -> fieldName,
+        "previousValue"   -> previousValue,
+        "newValue"        -> newValue
+      )
+      val recipients                              = Set("admin@example.com", "dev@example.com", "ri@example.com")
+      val expectedRequest: SendEmailRequest       = SendEmailRequest(recipients, "apiChangeOfApplicationDetails", expectedParameters)
+      emailWillReturn(expectedRequest)
+
+      await(connector.sendChangeOfApplicationDetails(requesterName, applicationName, fieldName, previousValue, newValue, recipients))
```

Comment:
I've just noticed that none of the tests in here assert/verify anything, are we just checking that the connector doesn't throw an exception?

Reply:
Yes I guess it will fail if you try to send the wrong json format, however I've just checked that if you comment out the code in email connector that actually sends the email all the tests still pass!